src/batched/KokkosBatched_trmm_serial_impl.hpp

```diff
@@ -0,0 +1,288 @@
+/*
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 3.0
+//       Copyright (2020) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+// 1. Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright
+// notice, this list of conditions and the following disclaimer in the
+// documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the Corporation nor the names of the
+// contributors may be used to endorse or promote products derived from
+// this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY NTESS "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL NTESS OR THE
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+// LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+// NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Questions? Contact Siva Rajamanickam (srajama@sandia.gov)
+//
+// ************************************************************************
+//@HEADER
+*/
+
+#ifndef __KOKKOSBATCHED_TRMM_SERIAL_IMPL_HPP__
+#define __KOKKOSBATCHED_TRMM_SERIAL_IMPL_HPP__
+
+#include "KokkosBatched_Util.hpp"
+#include "KokkosBatched_trmm_serial_internal.hpp"
```

Comment:
@e10harvey I suggest the file names follow the naming convention in the ```src/batched/``` directory. For example:  
```KokkosBatched_trmm_serial_internal.hpp``` --> ```KokkosBatched_Trmm_Serial_Internal.hpp```

```KokkosBatched_trtri_decl.hpp``` --> ```KokkosBatched_Trtri_Decl.hpp```

Reply:
@vqd8a thanks for the feedback. I renamed the files to match the naming convention. I see that the batched unit tests are currently disabled; would you still like to see those batched unit tests added?

Reply:
@e10harvey Thanks for fixing. The batched unit tests were enabled in #686. You should merge the ```develop``` branch to yours.